src/main/js/wiki/components/PageHeader.tsx

```diff
@@ -121,72 +111,72 @@ class PageHeader extends React.Component<Props, State> {
   };
 
   getPagePathWithoutRootDirectory(page, wiki) {
+    if (!page) {
+      return "";
+    }
+    if (page.path == undefined) {
+      return page;
+    }
     if (page.path.indexOf(wiki.directory) === 0) {
       return page.path.substring(wiki.directory.length + 1);
     }
-
     return page.path;
   }
-
   render() {
-    const { page, wiki, classes, onDelete, historyLink, branch, branches, pushBranchStateFunction } = this.props;
-
+    const {
+      page,
+      wiki,
+      classes,
+      onDelete,
+      historyLink,
+      inSettings,
```

Comment:
this fields have to be added to the properties

Reply:
- [x] done